test/e2e/type-module-interop/index.test.ts

```diff
@@ -12,10 +12,24 @@ describe('Type module interop', () => {
       files: {
         'pages/index.js': `
           import Link from 'next/link'
+          import Head from 'next/head'
```

Comment:
It seems this test suite is passing without the changes in this PR, are we missing an assertion to capture the previous failure?

Reply:
Weird, it failed if I comment out my patch on my machine.

<img width="819" alt="image" src="https://user-images.githubusercontent.com/3468483/165215252-16f4131e-04cf-4fd5-9cbd-020076d8e479.png">

Reply:
Oh was testing with the start test mode, does seem to fail without in dev mode 👍